src/main/java/org/traccar/config/Keys.java

```diff
@@ -1339,7 +1339,7 @@ private Keys() {
      */
     public static final ConfigKey<Boolean> REPORT_IGNORE_ODOMETER = new BooleanConfigKey(
             "report.ignoreOdometer",
-            List.of(KeyType.CONFIG),
+            List.of(KeyType.CONFIG, KeyType.DEVICE),
```

Comment:
We also need to add it to the web app.

Reply:
It's already here right?

https://github.com/traccar/traccar-web/blob/e5173d706c0eb463f86c3751d34745341f166c14/src/common/attributes/useCommonDeviceAttributes.js#L17